auth/Jenkinsfile

```diff
@@ -0,0 +1,33 @@
+podTemplate(label: 'buildSbtPod', containers: [
+  containerTemplate(name: 'maven', image: 'maven:3.3.9-jdk-8-alpine', ttyEnabled: true, command: 'cat'),
+  containerTemplate(name: 'docker', image: 'docker', ttyEnabled: true, command: 'cat'),
+  containerTemplate(name: 'helm', image: 'lachlanevenson/k8s-helm', ttyEnabled: true, command: 'cat')
+],
+volumes: [
+  hostPathVolume(mountPath: '/var/run/docker.sock', hostPath: '/var/run/docker.sock'),
+]) {
+   node('buildSbtPod') {
+     def repo = checkout scm
+     def gitCommit = repo.GIT_COMMIT
+     def gitBranch = repo.GIT_BRANCH
+     def shortGitCommit = "${gitCommit[0..10]}"
+
+     stage('Build and package Java Code') {
+       container('maven') {
+         sh "mvn -B clean package"
```

Comment:
This currently tries to build the base repo and fails. I changed it to `mvn -B -f ./auth/ clean package` and the `jar` built correctly. Also, in order to test it in keycloak we still need to package this jar into a docker container.